### ISSUE TITLE
where can I find the cost function and loss function in the code from AutoKeras technical paper?

### ISSUE THREAD

**Issue by CuriousSingularity** [keras-team/autokeras#1276]
I am trying to work on the low levels of the Autokeras. I would like to know where is the cost function and loss function defined in the code which were defined as the problem statements in the Autokeras technical paper.
And also where are the metrics passed to the Bayesian Optimization for tuning. 

Could anyone direct me in the code. Thank you so much.

**Comment by haifeng-jin** [comment]
You need to checkout the 0.4 version and lower. There is a bayesian optimization implemented. You can find the loss there. The 1.0+ is not following the paper anymore.

**Comment by CuriousSingularity** [comment]
Thanks for the reply. I got what I was looking for.
